src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -129,7 +134,9 @@ public String run(@Nonnull EnvVars launchEnv, @Nonnull String image, @CheckForNu
             argb.add("-e");
             argb.addMasked(variable.getKey()+"="+variable.getValue());
         }
-        argb.add("--entrypoint").add(entrypoint).add(image);
+        if (command != null) {
```

Comment:
`command` should be `@Nonnull`. If you wish to permit no specified commands (which I do not recommend, since AFAIK there is only one caller of this method, and it passes a command), then you would check `command.length > 0`, which would anyway be gratuitous since `add(String...)` would just be a no-op in that case.